config/samples/banzaicloud_v1beta1_kafkacluster.yaml

```diff
@@ -77,6 +77,83 @@ spec:
       # priorityClassName: "high-priority"
+
+      # Add custom log4j configurations
+      # Note: all these configurations are stored in /config/log4j.properties in the corresponding kafka pod
+      # log4jConfig: |
+        # log4j.rootLogger=DEBUG, stdout, kafkaAppender
+
+        # log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+        # log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+        # log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
+
+        # log4j.appender.kafkaAppender=org.apache.log4j.DailyRollingFileAppender
+        # log4j.appender.kafkaAppender.DatePattern='.'yyyy-MM-dd-HH
+        # log4j.appender.kafkaAppender.File=${kafka.logs.dir}/server.log
+        # log4j.appender.kafkaAppender.layout=org.apache.log4j.PatternLayout
+        # log4j.appender.kafkaAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
+
+        # log4j.appender.stateChangeAppender=org.apache.log4j.DailyRollingFileAppender
+        # log4j.appender.stateChangeAppender.DatePattern='.'yyyy-MM-dd-HH
+        # log4j.appender.stateChangeAppender.File=${kafka.logs.dir}/state-change.log
+        # log4j.appender.stateChangeAppender.layout=org.apache.log4j.PatternLayout
+        # log4j.appender.stateChangeAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
+
+        # log4j.appender.requestAppender=org.apache.log4j.DailyRollingFileAppender
+        # log4j.appender.requestAppender.DatePattern='.'yyyy-MM-dd-HH
+        # log4j.appender.requestAppender.File=${kafka.logs.dir}/kafka-request.log
+        # log4j.appender.requestAppender.layout=org.apache.log4j.PatternLayout
+        # log4j.appender.requestAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
+
+        # log4j.appender.cleanerAppender=org.apache.log4j.DailyRollingFileAppender
+        # log4j.appender.cleanerAppender.DatePattern='.'yyyy-MM-dd-HH
+        # log4j.appender.cleanerAppender.File=${kafka.logs.dir}/log-cleaner.log
+        # log4j.appender.cleanerAppender.layout=org.apache.log4j.PatternLayout
+        # log4j.appender.cleanerAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
+
+        # log4j.appender.controllerAppender=org.apache.log4j.DailyRollingFileAppender
+        # log4j.appender.controllerAppender.DatePattern='.'yyyy-MM-dd-HH
+        # log4j.appender.controllerAppender.File=${kafka.logs.dir}/controller.log
+        # log4j.appender.controllerAppender.layout=org.apache.log4j.PatternLayout
+        # log4j.appender.controllerAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
+
+        # log4j.appender.authorizerAppender=org.apache.log4j.DailyRollingFileAppender
+        # log4j.appender.authorizerAppender.DatePattern='.'yyyy-MM-dd-HH
+        # log4j.appender.authorizerAppender.File=${kafka.logs.dir}/kafka-authorizer.log
+        $ log4j.appender.authorizerAppender.layout=org.apache.log4j.PatternLayout
+        $ log4j.appender.authorizerAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
```

Comment:
**Question**: What is the reason we are using `$` in front of these lines instead of `#`?

Reply:
rookie mistake :face_in_clouds: